build/generate-docs.ts

```diff
@@ -448,9 +448,9 @@ function createExpressionsContent() {
             }
             content += `\n### ${key}\n`;
             content += `${value.doc}\n`;
-            if ('example' in value) {
-                content += `\n\`\`\`json\n"some-property": ${JSON.stringify(value.example)}\n\`\`\`\n`;
-            }
+            value.example.syntax.method.unshift(`"${key}"`);
+            content += `\nSyntax:\n\`\`\`js\n[${value.example.syntax.method.join(', ')}]: ${value.example.syntax.result}\n\`\`\`\n`;
+            content += `\nExample:\n\`\`\`json\n"some-property": ${JSON.stringify(value.example.value)}\n\`\`\`\n`;
```

Comment:
Factor the example generation out in a separate function.

Reply:
I tried that, it created a json that was less helpful, IMHO. Right now you can scroll horizontally, which I found to be better.
Another option is to change the example to be a string and then decide when to insert a new line, but I find it a bit too much...

Reply:
OK let me know when you are done with this, I have an idea I want to try. :)

Reply:
I've updated the code, feel free to hack.